_events/2016-04-02.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Hip Hop Hacks"
+description: "On April 2nd and 3rd, we will be partnering with The Young Hackers to put on our first hip hop hackathon! We’ll be exploring and creating projects at the intersection of hip hop, technology and education."
+date: 2016-04-02 12:00pm
+days:
+    - day: 2016-04-02
+      start: 11:30am
+      end: 8:00pm
+    - day: 2016-04-03
+      start: 10:30am
+      end: 6:00pm
+address: "45 W 18th St, New York, NY 10011"
+rsvp: "https://www.eventbrite.com/o/monthly-music-hackathon-nyc-2470452960"
+image: #"place-this-image-file-in-assets-slash-events.jpg"
+---
+
+# Schedule
+## Saturday, April 2, 2016
+- 11:30 AM Doors Open
+- 11:45 AM Lunch
+- 12:00 PM Talks by Special Guests
+- 1:30 PM Tech and Music Workshops
+- 6:30 PM Dinner
+- 7:00 PM Special Guest Performers (to be announced!)
+- 8:00 PM End
+
+## Sunday, April 3, 2016
+- 10:30 AM Doors Open
+- 12:00 PM Lunch
+- 3:00 PM Demos of Hacks
+- 5:00 PM Special Guest Performers (to be announced!)
+- 6:00 PM End
+
+# Talks
+
+- **Tony Prendatt Carter**
```

Comment:
I got really strange spacing when I put them on the new lines. I can definitely put it back that way (and get rid of the emphasis.) Want me to?